doc.ddoc

```diff
@@ -69,14 +69,14 @@ _=
 
 NAVIGATION_COMMUNITY=
 $(DIVC navblock,
-    $(UL $(TOCENTRIES
-        comparison,   D Feature List,
-        builtin,      Rationale for Builtins,
-        ctod,         Converting C to D,
-        cpptod,       Converting C++ to D,
-        pretod,       The C Preprocessor vs D,
-        cppcontracts, D Contract Programming vs C++
-    ))
+    $(ITEMIZE
+        $(A comparison,   D Feature List),
```

Comment:
There's an unrelated issue here - this should use e.g. comparison.html so we don't futz with two possible names for pages. I'll fix that.